commands/command_status.go

```diff
@@ -31,6 +33,9 @@ func statusCommand(cmd *cobra.Command, args []string) {
 	if porcelain {
 		porcelainStagedPointers(scanIndexAt)
 		return
+	} else if status_json {
```

Comment:
This `else` is redundant, since the above `if` branch terminates execution via a return. What do you think of:

```go
if porcelain {
        porcelainStagedPointers(scanIndexAt)
        return
}

if statusJson {
        jsonStagedPointers(scanIndexAt)
        return
}

// ...
```